platform.txt

```diff
@@ -24,6 +24,9 @@ tools.esp_ota.cmd.windows="{runtime.platform.path}/tools/espota.exe" -r
 tools.gen_esp32part.cmd=python3 "{runtime.platform.path}/tools/gen_esp32part.py"
 tools.gen_esp32part.cmd.windows="{runtime.platform.path}/tools/gen_esp32part.exe"
 
+tools.gen_insights_pkg.cmd = python3 {runtime.platform.path}/tools/gen_insights_package.py
+tools.gen_insights_pkg.cmd.windows = {runtime.platform.path}/tools/gen_insights_package.exe
```

Comment:
I suggest you quote and format the commands as the ones above in order to prevent possible path/etc issues.

Reply:
I get errors if I add quotes
```
/var/folders/0l/c75rh0953fng_6ccdgnm48gr0000gn/T/arduino_build_640628: -c: line 0: unexpected EOF while looking for matching `"'
/var/folders/0l/c75rh0953fng_6ccdgnm48gr0000gn/T/arduino_build_640628: -c: line 1: syntax error: unexpected end of file
```

Can you please help me out with this?

Reply:
@SuGlider please help with testing on Windows